src/i18n/en/translation.json

```diff
@@ -2270,7 +2271,20 @@
       "enable_rule_modal_description": "Enable the rule '{description}' identified by '{id}'?",
       "enable_rule_error": "Cannot enable rule",
       "no_rules_found": "No rules found",
-      "no_bypasses_found": "No bypasses found"
+      "no_bypasses_found": "No bypasses found",
+      "suppressed_alerts_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris laoreet lorem vel diam eleifend auctor. In massa dolor, finibus sit amet hendrerit nec, feugiat in sem.",
+      "error_loading_suppressed_alerts": "Cannot load suppressed alerts",
+      "no_suppressed_alerts": "No suppressed alerts found",
+      "add_suppressed_alert": "Suppress alert",
+      "suppressed_alert_description": "Description",
+      "suppressed_alert_id": "@:standalone.ips.rule_id",
```

Comment:
Is this string used?

Reply:
https://github.com/NethServer/nethsecurity-ui/blob/a08fdc9497fa0dbbec023e34fdb91fe357b8c70e/src/components/standalone/security/ips/IpsSuppressedAlerts.vue#L192